src/validation.cpp

```diff
@@ -5013,13 +5084,12 @@ bool LoadMempool(CTxMemPool& pool)
             if (amountdelta) {
                 pool.PrioritiseTransaction(tx->GetHash(), amountdelta);
             }
-            TxValidationState state;
             if (nTime > nNow - nExpiryTimeout) {
                 LOCK(cs_main);
-                AcceptToMemoryPoolWithTime(chainparams, pool, state, tx, nTime,
-                                           nullptr /* plTxnReplaced */, false /* bypass_limits */,
-                                           false /* test_accept */);
-                if (state.IsValid()) {
+                const MempoolAcceptResult result = AcceptToMemoryPoolWithTime(chainparams, pool, tx, nTime,
+                                                                                false /* bypass_limits */,
+                                                                                false /* test_accept */);
+                if (result.m_accepted) {
```

Comment:
Consider joining these lines:

```suggestion
                if (AcceptToMemoryPoolWithTime(chainparams, pool, tx, nTime,
                                               false /* bypass_limits */,
                                               false /* test_accept */).m_accepted) {
```